styles.css

```diff
@@ -1238,3 +1161,128 @@ body {
     translate: 10px;
   }
 }
+
+/* MOBILE MENU */
+
+header {
+  background-color: transparent;
+  width: 372px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 40px;
+  margin-left: 820px;
+}
+
+.head {
+  font-size: 1.2em;
+  font-weight: 700;
+}
+
+.nav-link {
+  transition: 0.7s ease;
+  color: black;
+}
+
+.nav-link:hover {
+  color: rgb(45, 245, 245);
+}
+
+.nav-icon {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 20px;
+  height: 3px;
+  margin: 5px auto;
+  margin-right: 10px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+@media (max-width: 768px) {
+  .header {
+    width: 332px;
+  }
+
+  .nav-icon {
+    display: block;
+  }
+
+  li a {
```

Comment:
- [ ] having made a great effort working on the functionality of the mobile menu when I click on the `hamburger`, the mobile menu links appear, and when I click on the `(X)`, the mobile menu links disappear and the links direct me to the necessary page in the webpage. But currently, I notice some wrong alignments of the `mobile version` of the project as it is in the Figma design. Kindly check the `mobile version` of your project on the `screen size of 375px` and make it align well with the screen

Your Design 👇 

![bel-mob](https://user-images.githubusercontent.com/78634841/209030670-1a423e12-3d96-4026-b64c-2d9c01ab1e8c.png)

Figma Design 👇 

![real-mob](https://user-images.githubusercontent.com/78634841/209030880-ac46befa-6e7e-4c96-bb10-f18bf4346f54.png)

Reply:
@brainconnect93 Good day. First of all, congratulations on your award yesterday!! I have tried my best to get rid of the extra space but couldn't. Please, can you help me with a way out?